packages/@storefront/structure/test/unit/filtered-list.ts

```diff
@@ -86,7 +87,7 @@ suite('FilteredList', ({ expect, spy, stub }) => {
 
   describe('onKeyDown()', () => {
     it('should exit early if the key is not Enter', () => {
-      const keyboardEvent: any = { keyCode: 1970 };
+      const keyboardEvent: any = { key: 1970 };
```

Comment:
Make this a string. Perhaps use one of the existing ones.

Reply:
Implemented. Resolving.